CairoMakie/src/primitives.jl

```diff
@@ -453,6 +450,25 @@ function draw_glyph_collection(scene, ctx, position, glyph_collection, rotation,
     return
 end
 
+struct CairoGlyph
+    index::Culong
+    x::Cdouble
+    y::Cdouble
+end
+
+function show_glyph(ctx, glyph::Culong, x, y)
+    cg = Ref(CairoGlyph(5930, x, y))
```

Comment:
```suggestion
function show_glyph(ctx, glyph, x, y)
    cg = Ref(CairoGlyph(glyph, x, y))
```
The type info is not needed directly, it gets converted anyway by `ccall` (and for some reason I sometime get an `Int64` reaching this function).